live-tests/README.adoc

```diff
@@ -51,11 +51,21 @@ To use this:
               $(which cargo-nextest) \
               root@YOUR_SCRIMLET_GZ_IP:/zone/oxz_switch/root/root
 
-3. On that system, unpack the tarball with:
+3. Copy omdb as well.
+
+     e.g., scp \
+              /home/dap/omicron-work/target/debug/omdb \
+              root@YOUR_SCRIMLET_GZ_IP:/zone/oxz_switch/root/root
+
+4. On that system, enable target blueprint:
```

Comment:
Could you move this to outside the terminal output (say, up to L22)?  I'd also add a little more context.  Maybe something like:

> The live tests require that the system's target blueprint is enabled.  This is to avoid a case where the live tests generate blueprints based on a target blueprint that is _not_ current and then wind up making a bunch of unrelated changes to the system.  On a fresh system, you will have to enable the target blueprint yourself:
>
>  `omdb --destructive nexus blueprints target enable current`
>
> This essentially enables reconfigurator, causing it to constantly try to make the system match its target blueprint.  You only need to do this once in the lifetime of the system, not every time you re-run the live tests.